widgetssdk/src/main/java/com/glia/widgets/core/survey/domain/GliaSurveyAnswerUseCase.java

```diff
@@ -80,4 +83,23 @@ public void validate(QuestionItem item) throws SurveyValidationException {
             }
         }
     }
+
+    public void trim(@Nullable List<QuestionItem> questions) {
+        if (questions == null) {
```

Comment:
this null check appears in many places in the usecase, maybe can have that check once, somewhere above before invoking that method?